influxdb/server.sls

```diff
@@ -25,8 +25,19 @@ influxdb_service:
   service.running:
   - enable: true
   - name: {{ server.service }}
+  {%- if server.admin_user is defined %}
+  - init_delay: 5
+  {%- endif %}
   - watch:
     - file: influxdb_config
     - file: influxdb_default
 
+{%- if server.admin_user is defined %}
+influxdb_create_admin_user:
+  cmd.run:
+  - name: curl -s -POST http://{{ server.admin.bind.address }}:8086/query --data-urlencode "q=CREATE USER {{ server.admin_user.name }} WITH PASSWORD '{{  server.admin_user.password }}' WITH ALL PRIVILEGES"
```

Comment:
Out of curiosity does InfluxDB tell you if the user you're trying to create already exist? I am trying to determine if we can use "stateful: True" and make the command return its state.